app/schemas/models/user.coffee

```diff
@@ -117,7 +117,7 @@ _.extend UserSchema.properties,
   mailChimp: {type: 'object'}
   hourOfCode: {type: 'boolean'}
   hourOfCodeComplete: {type: 'boolean'}
-  lastIP: {type: 'string'}
+  lastIP: {type: 'string'}    #TODO: deprecated, remove this from schema after updating geo location for existing users
```

Comment:
Don't need to change here, but we usually put a space between hash and TODO: `# TODO: blah blah`